packages/lib/server/getDefaultLocations.ts

```diff
@@ -22,7 +22,8 @@ export async function getDefaultLocations(user: User): Promise<EventTypeLocation
       (app) => app.slug === defaultConferencingData.appSlug
     )[0]; // There is only one possible install here so index [0] is the one we are looking for ;
     const locationType = foundApp?.locationOption?.value ?? DailyLocationType; // Default to Daily if no location type is found
-    return [{ type: locationType, link: defaultConferencingData.appLink }];
+    const credentialId = foundApp.credential.id ?? undefined;
+    return [{ type: locationType, link: defaultConferencingData.appLink, credentialId }];
```

Comment:
I don't really see changes in this PR that use this newly added `credentialId` prop.

Reply:
@SomayChauhan made those changes in https://github.com/calcom/cal.com/pull/15376#pullrequestreview-2107355807

> added a small fix in https://github.com/calcom/cal.com/commit/20fdb50081ae684723a5af78436068c0b09509a4
where the default event-types we create for the new user were not picking up the default app we just set.

Reply:
![334563703-b13e0b85-517f-42cc-8839-75cad8422823](https://github.com/calcom/cal.com/assets/32706411/534cf40a-0364-47ea-a0ec-0ff2bafd003b)
the reason I added `credentialId` was that in some cases the Location dropdown remained empty
although I cant replicate it now, so we can remove it too

Reply:
Yeah, let's remove that and maybe fix in a separate PR once we are able to replicate it.